soc/arm/nxp_s32/s32ze/Kconfig.series

```diff
@@ -14,5 +14,6 @@ config SOC_SERIES_S32ZE_R52
 	select VFP_DP_D16
 	select PLATFORM_SPECIFIC_INIT
 	select SOC_FAMILY_S32
+	select ARM_CUSTOM_REBOOT
```

Comment:
pls remove this line, custom reboot not yet supported on this soc, building `nxp_s32/common/power_soc.c` is guarded for now to build for s32k only.

Reply:
thanks, fixed